packages/clerk-js/src/ui/primitives/Text.tsx

```diff
@@ -23,7 +20,6 @@ const { applyVariants, filterProps } = createVariants(theme => {
         danger: { color: theme.colors.$danger500 },
         success: { color: theme.colors.$success500 },
         neutral: { color: theme.colors.$colorTextSecondary },
```

Comment:
Can we have `primary` -> `body` and `neutral` -> `secondary` ? I think that would be cleaner.

Reply:
Good call! I will add a new item to refactor this later.